### ISSUE TITLE
Error with `pin-project-lite:0.1.9`

### ISSUE THREAD

**Issue by nacardin** [http-rs/async-sse#10]
Seeing the following error after running `pin-project-lite` released version 0.1.9.
```
error[E0453]: allow(explicit_outlives_requirements) overruled by outer forbid(rust_2018_idioms)
  --> /home/ubuntu/.cargo/registry/src/github.com-1ecc6299db9ec823/async-sse-4.0.0/src/encoder.rs:9:1
   |
9  | / pin_project_lite::pin_project! {
10 | |     /// An SSE protocol encoder.
11 | |     #[derive(Debug)]
12 | |     pub struct Encoder {
...  |
17 | |     }
18 | | }
   | |_^ overruled by previous forbid
   | 
  ::: /home/ubuntu/.cargo/registry/src/github.com-1ecc6299db9ec823/async-sse-4.0.0/src/lib.rs:34:11
   |
34 |   #![forbid(rust_2018_idioms)]
   |             ---------------- `forbid` level set here
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0453]: allow(explicit_outlives_requirements) overruled by outer forbid(rust_2018_idioms)
  --> /home/ubuntu/.cargo/registry/src/github.com-1ecc6299db9ec823/async-sse-4.0.0/src/lines.rs:11:1
   |
11 | / pin_project! {
12 | |     /// A stream of lines in a byte stream.
13 | |     ///
14 | |     /// This stream is created by the [`lines`] method on types that implement [`BufRead`].
...  |
28 | |     }
29 | | }
   | |_^ overruled by previous forbid
   | 
  ::: /home/ubuntu/.cargo/registry/src/github.com-1ecc6299db9ec823/async-sse-4.0.0/src/lib.rs:34:11
   |
34 |   #![forbid(rust_2018_idioms)]
   |             ---------------- `forbid` level set here
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by Licenser** [comment]
hitting the same with pin project 0.1.10

**Comment by Licenser** [comment]
Found it is fixed in 4.0.1, for me running `cargo update -p async-sse` fixed this :)

**Comment by jbr** [comment]
closing this as resolved, please reopen if that's inaccurate
